test/JDBC/input/column_privileges-vu-prepare.sql

```diff
@@ -0,0 +1,35 @@
+use master;
+go
+
+create database column_privileges_vu_prepare_db;
+go
+
+use column_privileges_vu_prepare_db;
+go
+
+create table column_privileges_vu_prepare_tb1(arg1 int, arg2 int);
+go
+
+create table column_privileges_vu_prepare_tb2(arg3 int, arg4 int);
+go
+
+create login column_privileges_vu_prepare_log with password = 'EmptyPassword#';
+go
+
+create user column_privileges_vu_prepare_user for login column_privileges_vu_prepare_log;
```

Comment:
CREATE USER and GRANT statement are not supported until Babelfish 13.6. I'll suggest move this part to verify file.

Reply:
Updated in latest commit.